### PR TITLE
BACKPORT 4.1: Debian 9 & 10 on SUMA (#684)

### PR DESCRIPTION
**Pull request by Loquacity**
* debian suma setup

* changelog

* product/channel names

# Description

Short summary of why you created this PR

# Target branches

Which documentation version does this PR apply to?

- [ ] Master (Default)
- [x] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Review by keichwa — APPROVED** [review]

File: modules/client-configuration/pages/clients-debian.adoc [hunk 1]
```diff
@@ -41,6 +41,42 @@ apt install apt-transport-https
...
+
+The products you need for this procedure are:
+
+[[debian-channels-wizard]]
+[cols="1,1", options="header"]
+.Debian Products - WebUI
+|===
+| OS Version  | Product Name
+| {debian} 9  | SUSE Linux Enterprise Client Tools Debian 9 amd64
+| {debian} 10 | SUSE Linux Enterprise Client Tools Debian 10 amd64
```
Comment (mcalmer) [inline modules/client-configuration/pages/clients-debian.adoc:56]: The Client Tools are just a child product. Please check how you described Ubuntu 20.04. Both Debian versions are setup the same way.

**Review by mcalmer — CHANGES_REQUESTED** [review]
I think this can be done better. I do not have an up to date version of SUMA 4.1 docs to compare Ubuntu 20.04. But I think we described it there a bit different. Please check

**Review by jcayouette — APPROVED** [review]

**Comment by Loquacity** [comment]
Hey Michael, this is a backport so let's not make changes here. I'll merge this one, and make a note to take another look when I get back from leave.

**Comment by Loquacity** [comment]
> I think this can be done better. I do not have an up to date version of SUMA 4.1 docs to compare Ubuntu 20.04. But I think we described it there a bit different. Please check

Hey Michael, this is a backport so let's not make changes here. I'll merge this one, and make a note to take another look when I get back from leave.

**Pull request merged**